src/plutus-core-syntax.k

```diff
@@ -22,7 +22,8 @@ module PLUTUS-CORE-SYNTAX
   // expression as that in the Plutus Core specification.
 
   /* TODO: Make sure that K's built-in Float is compatible with PlFloat*/
```

Comment:
Comment should be updated. Maybe `TODO: add hook to ensure all floats are always of the correct format`